src/Arcus.WebApi.Tests.Projects.AzureFunction/HttpTriggerFunction.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Arcus.Observability.Correlation;
+using Arcus.WebApi.Logging.Correlation;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Arcus.WebApi.Tests.Projects.AzureFunction
```

Comment:
Nit - Maybe this is better name?
```suggestion
namespace Arcus.WebApi.Tests.Runtimes.AzureFunction
```

Reply:
Hmm, I don't find that more clearly. 'Runtimes' makes me think of not 'complete' projects while in fact it is a fully functional project used in the test suite.

Reply:
It feels to me since we are running tests against the Functions runtime but it's ok. Project doesn't say a thing if you ask me if you open up the solution, but that's just my opinion

Reply:
Ok, I'll change it.